tiny_skia/src/lib.rs

```diff
@@ -51,8 +52,9 @@ pub struct TinySkiaRenderer<W> {
     cache_color: CacheColor,
 
     image_cache: HashMap<Vec<u8>, (CacheColor, Rc<Pixmap>)>,
+    // The `u32` is a color encoded as a u32 so that it is hashable and eq.
     #[allow(clippy::type_complexity)]
-    glyph_cache: HashMap<(CacheKey, Color), (CacheColor, Option<Rc<Glyph>>)>,
+    glyph_cache: HashMap<(CacheKey, u32), (CacheColor, Option<Rc<Glyph>>)>,
```

Comment:
With `color`, the `Color` (really `AlphaColor<Srgb>`) is using `f32`, so it isn't `Hash` or `Eq`. We have a way to work around this with a wrapper type, but since here we're just dropping down to u8 colors anyway, I thought this might be okay here. I'm not sure though.